features/support/world.ts

```diff
@@ -83,16 +85,20 @@ export class World {
     } else {
       const stdout = new PassThrough()
       const stderr = new PassThrough()
-      const cli = new Cli({
-        argv: args,
-        cwd,
-        stdout,
-        stderr,
-        env,
-      })
+      const environment: IRunEnvironment = { cwd, stdout, stderr, env }
+      const { options, configuration: argvConfiguration } =
+        ArgvParser.parse(args)
+      const { runConfiguration } = await loadConfiguration(
+        {
+          file: options.config,
+          profiles: options.profile,
+          provided: argvConfiguration,
+        },
+        environment
+      )
       let error: any
       try {
-        const { success } = await cli.run()
+        const { success } = await runCucumber(runConfiguration, environment)
         if (!success) {
           error = new Error('CLI exited with non-zero')
```

Comment:
Should this now say something like "runCucumber threw an error"?

Reply:
Updated